changelog/Microsoft.IGAELM.json

```
@@ -1,5 +1,47 @@
 ﻿{
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "f0b1d404-83b8-4796-ac5f-748597e9df5e",
+          "ApiChange": "Relationship",
+          "ChangedApiName": "assignmentPolicies",
+          "ChangeType": "Addition",
+          "Description": "Added the **assignmentPolicies** relationship to [accessPackage](https://docs.microsoft.com/en-us/graph/api/resources/accessPackage?view=graph-rest-1.0) resource.",
```

Changelog generation bug
```suggestion
          "Description": "Added the **assignmentPolicies** relationship to the [accessPackage](https://docs.microsoft.com/en-us/graph/api/resources/accessPackage?view=graph-rest-1.0) resource.",
```